### ISSUE TITLE
Polices that import data are not correctly parsed by fregot

### ISSUE THREAD

**Issue by ahmedtalhakhan** [fugue/fregot#238]
While testing fregot, I came across this weird issue that any policy which tries to import something from the `data` section gives parsing errors in fregot.

Example policy

```
package test.policy

import input
import data.data_label

match[policy.name] {
   policies := data_label[input.cutomer_id]
   policy := policies[_]
   policy.name == input.name
}
```

When you try to load this policy to fregot, it complains with errors of type

```
repl% :load basic.rego
Loading basic.rego...
fregot (compile error):
  "basic.rego" (line 9, column 4):
  unknown variable:

    9|    policy := data_label[input.cutomer_id]
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  Undefined variable: data_label

```

The problem goes away if you comment out the `data` import statement and use the `data` variable directly. Below is the edited version which works with fregot

```
package test.policy

import input

match[policy.name] {
   policies := data.data_label[input.cutomer_id]
   policy := policies[_]
   policy.name == input.name
}
```

Although this works, but this is not good for testing policies with fregot.  OPA/rego policies frequently use the `data` section to import stuff. fregot should correctly interpret such imports. Otherwise, the policy needs to be modified every time someone needs to use fregot.

**Comment by jaspervdj-luminal** [comment]
Thanks a lot for reporting this @ahmedtalhakhan!

The issue was specifically around `import data.foo` where `foo` is a rule or variable
rather than a package.  This should be fixed in `fregot-v0.12.2`.

**Comment by ahmedtalhakhan** [comment]
Great! Thanks for responsiveness!
